src/library/Uno.Toolkit.Cupertino/Styles/Controls/SlidingSegmentedControl.xaml

```diff
@@ -406,17 +408,16 @@
 			<Setter.Value>
 				<DataTemplate>
 					<toolkit:ElevatedView Width="{Binding RelativeSource={RelativeSource Mode=TemplatedParent}, Path=TemplateSettings.SelectionIndicatorWidth}"
-										  Height="28"
+					                      Margin="{StaticResource CupertinoSlidingSegmentedItemMargin}"
 										  Background="Transparent"
-										  CornerRadius="8"
+										  CornerRadius="{CustomResource CupertinoSlidingSegmentedItemCornerRadius}"
```

Comment:
```suggestion
										  CornerRadius="{StaticResource CupertinoSlidingSegmentedItemCornerRadius}"
```

Reply:
Thanks, I didn't noticed it was set as `CustomResource`